drake/core/test/pendulum.h

```diff
@@ -0,0 +1,64 @@
+#ifndef DRAKE_CORE_TEST_PENDULUM_H_
+#define DRAKE_CORE_TEST_PENDULUM_H_
+
+#include <iostream>
+#include <cmath>
+
+#include "drake/core/Core.h"
```

Comment:
None of the drake content of this include is used. Consider replacing with '#include <Eigen/Dense>'